### PR TITLE
Depth of nested list

### PR DESCRIPTION
**Pull request by Saga305**
Added a code to verify the depth of the nested loop.

### CODE REVIEW

**Comment by Saga305** [comment]
@sagynangare could you please review this?

**Review by sagynangare — COMMENTED** [review]
is this worked for you??

**Comment by Saga305** [comment]
@sagynangare yes

**Comment by sagynangare** [comment]
I have to try it on my machine, let you know, wait

**Comment by Saga305** [comment]
@sagynangare sure, please

_Sent from my Redmi Note 3 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by sagynangare** [comment]
Hey, It is working properly but can you find is there any alternative solution will be for this instead of converting list into string and vice versa?

**Comment by Saga305** [comment]
@sagynangare :+1: 
Separate PR is created for this.
https://github.com/Saga305/Python/pull/3/

File: depth.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#Find depth of given number in nested list.
+print ("Enter list:")
+l = [x for x in list(input()) if x != ',']
+print ("Enter number:")
+v = int(input())
+st="l["
+c = 0
+ob = 0
```
Comment (Saga305) [inline depth.py:8]: ob is not used any where.

File: depth.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+cb = 0
```
Comment (Saga305) [inline depth.py:9]: same as above

**Comment by Saga305** [comment]
closing PR as code is not relevant.

**Pull request closed without merging**